tools/text_processing/text_processing/easyjoin.xml

```diff
@@ -53,6 +54,7 @@
                 </valid>
             </sanitizer>
         </param>
+        <param name="temporary_directory" type="text" value="/tmp" label="GNU sort's --temporary-directory option (can be changed to avoid 'No space left on device' error)."/>
```

Comment:
I don't think such an option should be exposed to the user. In theory, a user can enter here anything and Galaxy will try to write there ... this will also never be cleaned up.

Reply:
ok. As a quick fix I had set -T option(with our /workspace/tmp dir that can handle large files) in the `<command>` bloc. So I can do that again (but each time this tool will be updated, I will have to apply the patch, that's why I prefer creating PR)

Is the perl program update fine to you?

> this will also never be cleaned up

No, it's cleaned up I think. I've tried in my home dir, without `--debug` option the temp files are removed by `easyjoin`

Reply:
See my other comment. I think you can easily work-around this by setting the env variable for this tool at the admin level.

So if your /tmp is too small just re-set $TEMP to /var/tmp or something similar and it should work I assume. This is also what we do on your cluster by default.